src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -77,16 +77,13 @@ class WorkspaceReimburseView extends React.Component {
     }
 
     getRateDisplayValue(value) {
-        const numValue = parseFloat(value);
-        if (Number.isNaN(numValue)) {
-            return '';
-        }
-
-        return numValue.toFixed(3);
+        return value.toString().replace('.', this.props.fromLocaleDigit('.'));
     }
 
     setRate(value) {
-        const isInvalidRateValue = value !== '' && !CONST.REGEX.RATE_VALUE.test(value);
+        const decimalSeparator = this.props.fromLocaleDigit('.');
```

Comment:
use of `fromLocaleDigit` is wrong here as per your changes. Why? 

fromLocaleDigit accepts a localized char, `.` is not decimal in spanish so it should `this.props.fromLocaleDigit(',');`

Reply:
> use of `fromLocaleDigit` is wrong here as per your changes. Why?


Thanks, that is a good catch. They return the same value when passing `.` and I missed it and used the wrong function on `IOUAmountPage`, while using the right one on `BigNumberPad`. I'll open a PR with fix for this after the discussion around the `BigNumberPad`.

Reply:
`.` is the grouping separator in Spanish, i.e `,` in English.

Easy to miss, thanks @parasharrajat! 

Need to update this @eVoloshchak!

cc: @sketchydroide

Reply:
> Need to update this @eVoloshchak!

I can fix this in https://github.com/Expensify/App/pull/10028, would that be ok?

Reply:
Please hold on to it. We are first trying to decide why are these changes needed at all.